src/analyze/analyze.c

```diff
@@ -1139,22 +1154,51 @@ static int graph_one(sd_bus *bus, const UnitInfo *u, char *patterns[], char *fro
         assert(u);
 
         if (IN_SET(arg_dot, DEP_ORDER, DEP_ALL)) {
-                r = graph_one_property(bus, u, "After", "green", patterns, from_patterns, to_patterns);
+                r = graph_one_property(bus, u,
+                                       unit_dependency_to_string(UNIT_AFTER),
+                                       dependency_color[UNIT_AFTER],
+                                       dependency_style[UNIT_AFTER],
+                                       patterns, from_patterns, to_patterns);
                 if (r < 0)
                         return r;
         }
 
         if (IN_SET(arg_dot, DEP_REQUIRE, DEP_ALL)) {
-                r = graph_one_property(bus, u, "Requires", "black", patterns, from_patterns, to_patterns);
-                if (r < 0)
-                        return r;
-                r = graph_one_property(bus, u, "Requisite", "darkblue", patterns, from_patterns, to_patterns);
+                for (int dependency = UNIT_REQUIRES; dependency <= UNIT_PART_OF; ++dependency) {
```

Comment:
use `UnitDependency` as type here, not `int`

Reply:
done